js/tools.js

```diff
@@ -706,6 +706,18 @@ function getStorageAvailable() {
 	}
 }
 
+function refreshTotalCombinations(sign, number) {
```

Comment:
Missing JSDoc for this function.

Reply:
Functions should be declared as named function expressions (see [Point 7.1](https://github.com/airbnb/javascript#functions--declarations))

(but using `var` instead of `const` since this is a legacy file)